monai/apps/deepgrow/transforms.py

```diff
@@ -652,8 +652,12 @@ def bounding_box(self, points, img_shape):
 
     def __call__(self, data):
         d: Dict = dict(data)
+        image_key = first(self.key_iterator(d))
+        if image_key is None:
```

Comment:
technically `None` is a valid key:
```py
>>> import numpy as np
>>> from monai.transforms import ScaleIntensityd
>>> ScaleIntensityd(None)({None: np.zeros((1, 2,3))})
{None: array([[[0., 0., 0.],
        [0., 0., 0.]]], dtype=float32)}
>>> 

```

Reply:
Yes, I know that.
I think we discussed it before.
Anyway, maybe it's better to check the length of available keys, let me try later.

Thanks.

Reply:
Updated in the latest commit.

Thanks for the comments.